tests/devices/test_default_gaussian.py

```diff
@@ -696,7 +696,8 @@ def test_supported_gates(self, g, qop, gaussian_dev):
         @qml.qnode(gaussian_dev)
         def circuit(*x):
             """Reference quantum function"""
-            qml.Displacement(a, 0, wires=[0])
+            if "State" not in g:
+                qml.Displacement(a, 0, wires=[0])
```

Comment:
In tape mode, state preparations are now not allowed to be placed after unitary operations.